cmd/util.go

```diff
@@ -104,12 +104,23 @@ func slice2String(slice []string) string {
 }
 
 func genTargetCmd(cmd *cobra.Command, action, target string) bytes.Buffer {
-	var buf bytes.Buffer
-	executable, _ := cmd.Flags().GetString("executable")
-	buf.WriteString(executable + " " + action + " -target=" + target)
-	p, _ := cmd.Flags().GetInt("parallel")
-	buf.WriteString(fmt.Sprintf(" --parallelism=%d", p))
-	return buf
+  var buf bytes.Buffer
+  buf.WriteString("terraform " + action)
```

Comment:
Thanks @jmonfar 
It would be preferable not to override it. How about considering something like this.

```suggestion	
  executable, _ := cmd.Flags().GetString("executable")
  buf.WriteString(executable + " " + action)
```

Reply:
Of coure, it's better. My fix was for the original code without taking into account the other pending PR.